src/main/java/pl/project13/jgit/DescribeCommand.java

```diff
@@ -401,15 +384,9 @@ private String createMatchPattern() {
       return ".*";
     }
 
-    StringBuffer buf = new StringBuffer();
-    buf.append("^refs/tags/\\Q");
-    buf.append(matchOption.get().replace("*", "\\E.*\\Q").replace("?", "\\E.\\Q"));
-    buf.append("\\E$");
-    return buf.toString();
-  }
-
-  private void log(Object... parts) {
```

Comment:
could have remained but used the `mojo.getLog()` IMO, but not a big problem; I don't mind the current state